src/Execution/Arguments/ArgPartitioner.php

```diff
@@ -187,6 +188,10 @@ public static function methodReturnsRelation(
             return false;
         }
 
+        if (! class_exists($returnType->getName())) {
+            throw new Exception('Class '.$returnType->getName().' does not exist, did you forget to alias the Eloquent relation class?');
```

Comment:
```suggestion
            throw new Exception('Class '.$returnType->getName().' does not exist, did you forget to import the Eloquent relation class?');
```

Aliasing would be `use App\Foo as Bar;`